docs/BUILD.md

```diff
@@ -0,0 +1,41 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Release workflow](#release-workflow)
+  - [Step 1: Release](#step-1-release)
+  - [Step 2: Publish](#step-2-publish)
+  - [Checklist](#checklist)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Release workflow
+
+The main goal of the build process is to build and publish the docker image to the docker hub repo: 'commercetools/commercetools-project-sync'
+     
+## Step 1: Release
+
+Create a PR for the new release: 
+- Increment the release version to the new library version, please follow the [semantic versioning](https://semver.org/) for finding the new version.
+- Make sure all the dependencies are up-to-date everywhere in the documentation files and project files.
```

Comment:
We are not doing a dependency update manually as we are using dependabot, I would remove this as it's confusing what is meant by that.

Reply:
And what about the version upgrade, We have to update the new version in all the docs right?
Example: Readme has version number in docker run examples.

Reply:
Yes you are right, I forgot to write about it, you need to define versions but not the dependency 👍🏽

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/231/commits/84049690439cd99b6dfef78d6574343df665c3e0